bin/transform-gisaid

```diff
@@ -50,7 +76,7 @@ if __name__ == '__main__':
         formatter_class=argparse.RawTextHelpFormatter
     )
     parser.add_argument("gisaid_data",
-        default="s3://nextstrain-ncov-private/gisaid.ndjson.gz",
+        default="data/gisaid.ndjson.gz",
```

Comment:
this looks like a local adaptation. Any reason why we should not use the s3 file there ?